content/strategy-goals/strategy/code-graph/search/core.md

```diff
@@ -1,38 +1,46 @@
+We have worked toward the goal of indexing open source code to make our search universal:
+
+- The Sourcegraph Cloud global index has now 2.6M repositories representing all repositories with 5 stars or more from GitHub.com and GitLab.com. While working towards this goal, we have made changes that have yielded trickle-down benefits to all Sourcegraph deployments, for instance, we accomplished a [5x reduction](https://about.sourcegraph.com/blog/zoekt-memory-optimizations-for-sourcegraph-cloud/) in the memory usage of Zoekt, our trigram-based indexed search backend.
+- We have made progress towards expanding code host coverage to include repositories from non-GitHub.com and GitLab.com code hosts: you can now search more than [34k repositories](https://sourcegraph.com/search?q=context:global+r:%5Esrc%5C.fedoraproject%5C.org/+type:repo+count:all&patternType=literal) from src.fedoraproject.org on sourcegraph.com.
+- We have released the beta version of [dependencies search](https://docs.sourcegraph.com/code_search/how-to/dependencies_search) enabling search through the dependencies of your repositories for Go and npm.
+
+We conducted discovery work to better understand the bottlenecks of our search infrastructure on large monorepos (those with a working directory > 6GB) and started work on some prototypes to address our customers' most pressing needs. We established a baseline and created a synthetic gigarepo to measure progress towards our goal. Our synthetic gigarepo is a representative monorepo with a HEAD working copy size of 15GB.
+
+As a first step towards improving the ranking of results, we have enabled [zoekt results ranking](https://docs.sourcegraph.com/dev/background-information/architecture/indexed-ranking#result-ranking) to further rank results within repositories.
 
 ## What's next and why
 
-### FY23Q1
+### FY23Q2
 
 #### Goals
 
-- **Monorepo performance**: At a P75 level, gigarepo will index in less than 30 minutes, indexed searches complete in < 2s and unindexed searches complete in < 10s. Gigarepo is a representative monorepo with a HEAD working copy size of 15GB.
-- **Ranking**: We start tracking ranking quality, using selected search results as a proxy.
-- **Code host coverage**: Sourcegraph Cloud indexes public repositories globally from the most popular package hosts.
+- **Monorepo performance**: At a P75 level, synthetic gigarepo will index in less than 30 minutes, indexed searches will complete in < 2s and unindexed searches will complete in < 10s.
```

Comment:
Maybe worth explaining what a gigarepo is here, or using another term that more people will understand.

Reply:
A direct link could work too